sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/logging/JsonLoggingMdcTest.java

```diff
@@ -84,23 +83,6 @@ void shouldLogJsonWithMdcInRequestContext(CapturedOutput capturedOutput) {
                     .isNotEmpty());
   }
 
-  @Test
-  @Disabled("No trace token support implemented yet: PLP-655")
-  void shouldLogJsonWithTraceTokenInMdc(CapturedOutput capturedOutput) {
```

Comment:
Could this test be enabled instead of being removed?

Reply:
Yeah, but this exact behaviour is also tested in the TraceTokenTests and and I would rather keep tests that explicitly cover trace tokens in one class.